Pages/itemPage.js

```diff
@@ -0,0 +1,55 @@
+class ItemPage {
```

Comment:
Is there any page objects relations you noticed ?
For example, does it make sense to inherit page object "HomePage"

Reply:
I added inheritance from Page class.